setup.py

```diff
@@ -21,7 +21,8 @@
               'bluesky.callbacks'],
     package_data={'bluesky': ['schema/*.json']},
     install_requires=['jsonschema', 'traitlets', 'prettytable', 'cycler',
-                      'numpy', 'matplotlib', 'super_state_machine'],
+                      'numpy', 'matplotlib', 'super_state_machine',
+                      'historydict'],
```

Comment:
I do not think _any_ of these should be in the `install_requires` section, but should all be the `requires` section.  The `install_requires` is for stuff that is imported in `setup.py` (ex if you are build c-extensions with numpy you need to have numpy installed at install/build time to be able to use numpy to find it's headers (and to have the header)).